ddr/tools/getmacros

```diff
@@ -162,6 +163,7 @@ process_one_policy_file() {
 		-e '/@ddr_namespace:/p' \
 		-e '/^ *# *define /p' \
 		-e '/^ *# *undef /p' \
+		-e '\|^/\* # *undef |p' \
```

Comment:
I don't see how this can work; the pattern must be enclosed with two `/` characters.
Also we don't want to match spaces between `#` and `undef`.

Reply:
From https://www.gnu.org/software/sed/manual/sed.html#Regexp-Addresses

```
\%regexp%
(The % may be replaced by any other single character.)

This also matches the regular expression regexp, but allows one to use a different delimiter than /. 
```

Reply:
Thanks, I wasn't aware of that usage. I wouldn't be surprised it that's a GNU extension even though that page doesn't describe it as such. I think we should use `/` and escape as necessary.

Reply:
As far as I can tell, it seems like it is indeed an extension to the posix standard for sed: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sed.html

Interestingly, it seems the posix standard suggests but does not specify that sed uses `/` as a delimiter for addresses.

It looks like bsd sed supports this behavior: https://www.freebsd.org/cgi/man.cgi?query=sed&sektion=&n=1